packaging/fedora/pack-source

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -e
+
+show_help() {
+    echo "usage: $(basename "$0") [-v <version>] [-o <output-dir>] [-g] [-h]"
+    echo "       -v <version>       set version"
+    echo "       -o <output-dir>    write source packages to this directory"
+    echo "       -g                 use 'git describe' output as version string"
+    echo "       -s                 use single source archive instead of no-vendor and"
+    echo "                          only-vendor packages"
+    echo "       -h                 show help"
+    exit 1
+}
+
+if [ ! -e packaging/fedora/snapd.spec ]; then
+    echo "must be executed at the top of srcdir"
+    exit 1
+fi
+
+outdir=.
+single=0
+
+while getopts "v:o:gsh" arg; do
+    case "$arg" in
+        o)
+            outdir="$OPTARG"
+            ;;
+        v)
+            version="$OPTARG"
+            ;;
+        g)
+            version="$(git describe | tr '-' '.')"
+            ;;
+        s)
+            single=1
+            ;;
+        h|*)
+            show_help
+            ;;
+    esac
+done
+
+set -x
+
+if [ -z "$version" ]; then
+	# try to read if tform spec
+	version="$(awk '/^Version: / { print $2}' packaging/fedora/snapd.spec)"
```

Comment:
Hmm, this is hard-coding fedora spec in the world where we have an openSUSE spec as well. This feels wrong especially that we also use this script on openSUSE.

Reply:
As discussed on IRC, this can be dropped now.